src/PackagePathResolverInterface.php

```diff
@@ -8,7 +8,7 @@
  *
  * @author Nico Schoenmaker <nschoenmaker@hostnet.nl>
  */
-interface PackagePathResolver
+interface PackagePathResolverInterface
```

Comment:
Please note that this is a BC break. If you're not planning on bumping a major, please add a BC layer for this